tests/genai/judges/test_make_judge.py

```diff
@@ -366,3 +367,133 @@ def mock_invoke(model_uri, prompt, assessment_name):
     assert "Query: test" in captured_prompt
     assert "Response: answer" in captured_prompt
     assert "Custom: custom_value" in captured_prompt
+
+
+def test_judge_registration_as_scorer(tmp_path):
+    tracking_uri = "sqlite:///{}".format(tmp_path.joinpath("mlflow.db"))
```

Comment:
Is it necessary to set sqlite as the tracking URI explicitly here? Ditto in the test case below

(IIRC sql is the default for many test suites?)